src/Stratis.Bitcoin.Tests/P2P/PeerConnectorTests.cs

```diff
@@ -85,8 +84,8 @@ public void PeerConnectorAddNode_CanAlwaysStart()
             var nodeSettings = new NodeSettings();
 
             var connectionSettings = new ConnectionManagerSettings(nodeSettings);
-
-            var connector = new PeerConnectorAddNode(this.asyncLoopFactory, DateTimeProvider.Default, this.extendedLoggerFactory, this.network, this.networkPeerFactory, this.nodeLifetime, nodeSettings, connectionSettings, peerAddressManager, new SelfEndpointTracker());
+            var networkPeerFactory = new Mock<INetworkPeerFactory>();
```

Comment:
How come this changed?

Reply:
There was a global this.networkPeerFactory field in the class that wasn't used, so I implemented them in each test.